src/viewwindowmgr.cpp

```diff
@@ -142,11 +142,13 @@ ViewWindow * ViewWindowMgr::addNewTab( bool set_active )
 	if ( set_active || m_Windows.size() == 1 )
 		m_tabWidget->setCurrentWidget( tabdata.widget );
 	
+#if defined (USE_WEBKIT)
```

Comment:
Implementing the linkClicked signal is pretty simple and it allows you to use the same proven code for both WebKit and WebEngine. What is the advantage of your solution?

Reply:
I've implemented what's working for me. There is no other reason.